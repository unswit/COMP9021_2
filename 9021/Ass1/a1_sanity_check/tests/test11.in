Please convert 123 by using ABC
