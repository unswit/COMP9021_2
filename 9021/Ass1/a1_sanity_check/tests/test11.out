How can I help you? I don't get what you want, sorry mate!
